src/prefect/experimental/workers/base.py

```diff
@@ -36,6 +107,8 @@ def __bool__(self):
 @register_base_type
 class BaseWorker(abc.ABC):
     type: str
+    job_configuration: Optional[BaseJobConfiguration] = None
+    job_configuration_variables: Optional[BaseVariables] = None
```

Comment:
Similar comment regarding typing.
```suggestion
    job_configuration_variables: Optional[Type[BaseVariables]] = None
```